C/primitive/elements/jets.c

```diff
@@ -482,6 +482,42 @@ bool current_issuance_token_amt(frameItem* dst, frameItem src, const txEnv* env)
   return true;
 }
 
+/* tapleaf_version : ONE |- TWO^8 */
+bool tapleaf_version(frameItem* dst, frameItem src, const txEnv* env) {
+  (void) src; // src is unused;
+  write8(dst, env->taproot->leafVersion);
```

Comment:
Leaf version is actually TWO^7. Maybe keeping it TWO^8 makes using it easier. Just making sure that this is intentional

Reply:
X^n notation is only defined when n is a power of 2.  In turn, the (to be) standardized library functions are also only defined on TWO^n where n is a power of 2.

Reply:
Also worth mentioning it is the low bit that is always set to 0, not the high bit.  This is how _leaf version_ is defined in taproot.